### DISCUSSION TITLE
[dubbo-spi-extensions] Introduce bytebuddy proxy in apache/dubbo-spi-extensions

### DISCUSSION THREAD

**Discussion by songxiaosheng** [apache/dubbo#13831] [General]
Dubbo's default proxy is javassist. It has been tested by the practice of mass production.
Bytebuddy proxy can be chosen from dubbo-spi-exntensions. It still needs to be tested over time.
ref #13555

**Comment by AlbumenJ** [comment]
Track in https://github.com/apache/dubbo/issues/13867
